src/analyses/dependence_graph.h

```diff
@@ -38,23 +87,40 @@ class dep_graph_domaint:public ai_domain_baset
     ai_baset &ai,
     const namespacet &ns);
 
+  virtual void make_bottom()
+  {
+    has_values=tvt(false);
+  }
+
+  virtual void make_top()
+  {
+    has_values=tvt(true);
+  }
+
+  virtual void make_entry()
+  {
+    has_values=tvt::unknown();
+  }
+
```

Comment:
I think we should set all member variables to a specific value on ``make_bottom()/make_top()/make_entry()``. It's currently fine for all the analyses as we only call those methods on newly created states and don't use them in the actual analysis, but I think this would add some clarity.